src/queries/incremental-compilation.md

```diff
@@ -137,8 +137,10 @@ algorithm like so:
     it did not change.
 
 ## Resources
-The initial design document can be found at https://github.com/nikomatsakis/rustc-on-demand-incremental-design-doc/blob/master/0000-rustc-on-demand-and-incremental.md, which expands on the memoization details, provides more high-level overview and motivation for this system.
+The initial design document can be found at [here][initial-design], which expands on the memoization details, provides more high-level overview and motivation for this system.
```

Comment:
Also remove the `at`:
```suggestion
The initial design document can be found [here][initial-design], which expands on the memoization details, provides more high-level overview and motivation for this system.
```

Reply:
Can maybe remove the entire `here` link and link to the text?:

> The [initial design document][initial-design] expands on the memoization details, and  provides more high-level overview and motivation for this system.